recipes/libpcap/all/conanfile.py

```diff
@@ -98,6 +98,7 @@ def generate(self):
                 # to inject this compilation flag themselves
                 tc.variables["USE_STATIC_RT"] = False
             tc.cache_variables["DISABLE_DPDK"] = True
+            tc.cache_variables["CMAKE_DISABLE_FIND_PACKAGE_OpenSSL"] = True
```

Comment:
I'm wondering if we should have the variable `tc.variables["ENABLE_REMOTE"] = False` without any if-clause. What do you think?

Reply:
It seems to me that remote can be used, even without ssl cf https://github.com/the-tcpdump-group/libpcap/blob/master/CMakeLists.txt#L3171
So I think we should not change its logic.
A future PR could add openssl to the recipe dependencies if needed